### PR TITLE
date and time / it-academy.by / M-JC1-66-20 / Anastasia Gawriljuk / Done tasks 3/3

### PR DESCRIPTION
**Pull request by Semargl92**
## Pull request state

Actual status for PR:
- [x] implemented
- [x] reviewed
- [x] fixed
- [ ] approved

### CODE REVIEW

File: src/main/java/com/rakovets/course/java/core/practice/date_and_time/date_wrapper/DateWrapper.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        return Math.abs(DAYS.between(first, second));
+    }
+
+    public static LocalDate getDateAfterDays (LocalDate date, int days) {
+        TemporalAdjuster afterDays = temporal -> temporal.plus(days, DAYS);
+        return (LocalDate) afterDays.adjustInto(date);
+    }
+
+    public static TemporalAdjuster closestNewYear = temporal ->
+            (temporal.get(ChronoField.DAY_OF_YEAR) >= 183)
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/date_and_time/date_wrapper/DateWrapper.java:45]: What is 183?

**Review by rakovets — APPROVED** [review]
Perfect job!

**Review by rakovets — CHANGES_REQUESTED** [review]
Need to write test/demo for SPecification of tasks 2

**Comment by Semargl92** [comment]
> Need to write test/demo for SPecification of tasks 2

already have those tests, check, please: 
void getDateAfterDays
void adjustIntoClosestNewYear

**Review by rakovets — APPROVED** [review]
Approved.

**Pull request closed without merging**